### PR TITLE
Remove bullet for task and radio list items

### PR DESCRIPTION
**Pull request by nicholaschuayunzhi**
Resolves https://github.com/MarkBind/markbind/issues/158

image

Aligns the input to the text as well.

Site Preview:

image

### CODE REVIEW

File: asset/css/markbind.css [hunk 1]
```diff
@@ -16,3 +16,15 @@ blockquote {
         background-color: #fff;
     }
 }
+
+.radio-list-item,
+.task-list-item {
+    list-style-type: none;
+    margin: 0 0 0 -1em;
+}
```
Comment (acjh) [inline asset/css/markbind.css]: This block should be on line 5, based on increasing specificity (and taking into account alphabetical order).

File: asset/css/markbind.css [hunk 2]
```diff
@@ -16,3 +16,15 @@ blockquote {
...
+
+.radio-list-input,
+.task-list-item-checkbox {
+    height: 10px;
+    width: 10px;
+}
```
Comment (acjh) [inline asset/css/markbind.css]: Why is this block necessary?

Comment (nicholaschuayunzhi) [reply asset/css/markbind.css]: I added so that the text will be inline regardless of list type (radio, task, bullet).

If i remove it the example above will be:
image

Comment (acjh) [reply asset/css/markbind.css]: This reduces the size of a click target, which is undesirable.

Since we are already tweaking `margin` above, how about using `-1.2em` instead of `-1em`?

task, bullet, radio

Note that both ways are browser-dependent, so we shouldn't fret over this. If you do this:

```css
html, body {
    font-size: unset;
}
```

task, bullet, radio + unset

If you change the font size of your browser, the misalignment is even more apparent.

Even GitHub doesn't care:

- [ ] task
- bullet

Comment (nicholaschuayunzhi) [reply asset/css/markbind.css]: > Since we are already tweaking margin above, how about using -1.2em instead of -1em?

👍 Here's what it looks like (original). On its right is something i want to suggest which is adding
```css
.radio-list-input,
.task-list-item-checkbox {
    vertical-align: text-bottom;
}
```
It makes the input more aligned and the lists become more compact too.

image

Comment (acjh) [reply asset/css/markbind.css]: That misaligns the input on Safari 🤣

original | vertical-align: text-bottom;
-- | --
screenshot 2018-03-10 22 52 48 | screenshot 2018-03-10 22 53 34

File: asset/css/markbind.css [hunk 3]
```diff
@@ -2,17 +2,24 @@ blockquote {
...
+
     .navbar-nav .open .dropdown-menu {
         background-color: #fff;
     }
 }
+
+.radio-list-input,
+.task-list-item-checkbox {
+    vertical-align: text-bottom;
+}
```
Comment (acjh) [inline asset/css/markbind.css]: As [mentioned](https://github.com/MarkBind/markbind-cli/pull/35#discussion_r173624526), this misaligns the input on Safari. Let's do without this.

File: asset/css/markbind.css [hunk 4]
```diff
@@ -2,17 +2,24 @@ blockquote {
     font-size: 14px;
 }
 
-.radio-list-item label {
-    font-weight: inherit;
```
Comment (acjh) [inline asset/css/markbind.css]: Why was this removed?

Comment (nicholaschuayunzhi) [reply asset/css/markbind.css]: Apologies.. I was careless about removing the styling.

**Review by acjh — APPROVED** [review]

**Comment by acjh** [comment]
@nicholaschuayunzhi Can you resolve the conflicts?

**Comment by nicholaschuayunzhi** [comment]
@acjh updated!
